update.cf

```diff
@@ -10,17 +10,15 @@ body common control
                           "update_def",
                           "u_cfengine_enterprise",
                           @(u_cfengine_enterprise.def),
-                          "cfe_internal_dc_workflow",
-                          "cfe_internal_bins",
-                          "cfe_internal_update_policy",
-                          "cfe_internal_update_bins",
-                          "cfe_internal_update_processes",
+                          "mpf_update",
+
       };
 
       version => "update.cf $(update_def.current_version)";
 
       inputs => {
                   @(cfengine_update_controls.update_def_inputs),
+                  @(update_def.inputs),
```

Comment:
I was thinking that I needed to handle definition of empty lists, so I pulled `def.update_inputs` into `update_def.inputs`

https://github.com/cfengine/masterfiles/pull/730/files#diff-8c9a709a840b3d3b6c09f6d4d5e4def6R103